### DISCUSSION TITLE
Minor bug in video for d_control_flow_strings

### DISCUSSION THREAD

**Discussion by jlstanley-git** [CleanCut/ultimate_rust_crash_course#37] [General]
In the video, when you negated the written description of the end condition for the double() function ("until `x` is larger than 500"), you incorrectly wrote the while loop as:
  while x < 500
but the correct negation would be:
  while x <= 500
Yes, this is a minor issue in this context and doesn't change the result, but it's a detail that could cause problems if the end condition was a number like 512.

**Comment by jlstanley-git** [comment]
Duplicate to an almost identical prior comment in Udemy Q&A on the video that I didn't find before posting here.

**Comment by CleanCut** [comment]
👍🏻 This is fixed in the pending update to the course
